lib/dpul_collections/solr.ex

```diff
@@ -10,11 +10,22 @@ defmodule DpulCollections.Solr do
     response.body["response"]["numFound"]
   end
 
+  @query_field_list [
```

Comment:
we might want to push this to the solr config at some point; it will probably get long